covid-dashboard/src/js/getApiData.js

```diff
@@ -0,0 +1,81 @@
+// for table, list with flags
+async function getCountriesAndGlobalInfo() {
+  const urlCountries = 'https://disease.sh/v3/covid-19/countries';
```

Comment:
Could you please move the common part of url in separate variable and reuse between functions?